packages/mui-material/src/PaginationItem/PaginationItem.d.ts

```diff
@@ -14,23 +14,51 @@ export interface PaginationItemPropsSizeOverrides {}
 export interface PaginationItemPropsColorOverrides {}
 
 export interface PaginationItemSlots {
+  /**
+   * The component that renders the root slot.
+   * @default FirstPage
+   */
   first: React.ElementType;
+  /**
+   * The component that renders the last slot.
+   * @default LastPage
+   */
   last: React.ElementType;
+  /**
+   * The component that renders the next slot.
+   * @default NavigateNext
+   */
   next: React.ElementType;
+  /**
+   * The component that renders the previous slot.
+   * @default NavigateBefore
+   */
   previous: React.ElementType;
 }
 
 export type PaginationItemSlotsAndSlotProps = CreateSlotsAndSlotProps<
   PaginationItemSlots,
   {
-    first: SlotProps<React.ElementType<React.HTMLProps<HTMLElement>>, {}, PaginationItemOwnerState>;
-    last: SlotProps<React.ElementType<React.HTMLProps<HTMLElement>>, {}, PaginationItemOwnerState>;
-    next: SlotProps<React.ElementType<React.HTMLProps<HTMLElement>>, {}, PaginationItemOwnerState>;
-    previous: SlotProps<
-      React.ElementType<React.HTMLProps<HTMLElement>>,
-      {},
-      PaginationItemOwnerState
-    >;
+    /**
+     * Props forwarded to the first slot.
+     * By default, the available props are based on svg element.
+     */
+    first: SlotProps<'svg', {}, PaginationItemOwnerState>;
```

Comment:
I don't think it should be just `svg`. It can render any React element, button, or custom icon. The earlier version was correct.